test/js/eventemitter.js

```diff
@@ -21,6 +21,20 @@ describe('Verify EventEmitter', function() {
 
             thing.run(n)
         })
+
+        it('should remove listeners for the test event', function(done) {
+            let thing = new bindings.EmitterThing()
+            thing.on('test1', function(ev) { })
+            thing.on('test2', function(ev) { })
+
+            expect(thing.eventNames()).to.be.equal(['test2', 'test1'])
+            thing.removeAllListeners('test1')
+            expect(thing.eventNames()).to.be.equal(['test2'])
+            thing.on('test1', function(ev) { })
+            thing.removeAllListeners()
```

Comment:
I generally prefer 2 different behaviors to be split to 2 different tests, with state generation to be explicitly individual to each test

Reply:
fixed in e68a3341803f7c3e4752dba079452a93fa438122